tools/build-release.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/sh
+
+APP_VERSION=$(npm pkg get version | sed 's/"//g')
+
+echo ''
+echo '|============================================|'
+echo "  Preparing build v$APP_VERSION"
+echo '  Taken from the root ./package.json file.'
+echo '|============================================|'
+echo ''
+sleep 1
+echo 'Do you need to increment the save file version?'
+echo 'If you do, cancel/exit this script now.'
+echo 'Update the following file before running this script: '
+echo '> ./src/Constants.ts'
+echo ''
+sleep 2
+echo 'Do you want to include a changelog with this release?'
+echo 'If you do, cancel/exit this script now.'
+echo 'Update the following file before running this script: '
+echo '> ./src/Constants.ts'
+echo '> ./doc/source/changelog.rst'
+echo ''
+sleep 2
```

Comment:
Why waste 5 seconds every single time this is run?

Reply:
Actually 7, as there is one more `sleep` below

Reply:
Since this is done non-frequently, the sleep allows the publisher to cancel out of the build if he notices he should update the changelog or save migration number before running the command. IMO waiting 7 secs for something done once a day is fine.